src/charm.py

```diff
@@ -268,6 +268,33 @@ def _on_rotate_wordpress_secrets_action(self, event: ActionEvent):
         self._reconciliation(event)
         event.set_results({"result": "ok"})
 
+    def _update_database(self, dry_run: bool = False) -> types_.ExecResult:
```

Comment:
Nit: Can you reorder the function location, it should be below _on_update_database_action, see also https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#function-and-method-ordering

Reply:
Thanks! Done.